runtime/lua/vim/treesitter.lua

```diff
@@ -114,4 +114,24 @@ function M.get_string_parser(str, lang, opts)
   return LanguageTree.new(str, lang, opts)
 end
 
+--- Determines whether a node is the parent of another
+-- @param dest the possible parent
+-- @param source the possible child node
+function M.is_parent(dest, source)
```

Comment:
Probably `is_ancestor` would be more clear

Reply:
I think "parent" is the standard terminology for trees in CS?